src/app/loginComponents/accounts/internal/accounts.component.html

```diff
@@ -25,16 +25,25 @@ <h3>Dockstore Profile</h3>
       <mat-list>
         <mat-list-item>
           <h1 matLine>
-            <strong>Username:</strong>
+            <mat-icon>account_circle</mat-icon> <strong class="align-middle">Username:</strong>
```

Comment:
This is not specific to this PR, but I don't think we should have a colon here, especially with the name below being inside a `<p>`, which puts it on a separate line. I think we should follow [this guideline](https://ux.stackexchange.com/questions/3611/should-label-and-field-be-separated-with-colon), where we either have a colon and don't visually distinguish the label, or distinguish the label and don't have the colon.

Since there are several panels (GitHub and Google) with this pattern, so if the OICR folks agree(?), could you log this as a separate issue?

Reply:
I agree, I like the second option (distinguish the label and don't have the colon)

Reply:
Yeah, I like the second option more as well.

Reply:
ga4gh/dockstore#2055